spec/classes/agent_spec.rb

```diff
@@ -58,11 +58,12 @@
 
         it {
           should contain_file('sensu_agent_config').with({
-            'ensure'  => 'file',
-            'path'    => '/etc/sensu/agent.yml',
-            'content' => agent_content,
-            'require' => 'Package[sensu-go-agent]',
-            'notify'  => 'Service[sensu-agent]',
+            'ensure'    => 'file',
```

Comment:
there should be a test that demonstrates when `show_diff` is set to `false`.

Reply:
Test added